docs(dns): add docs for dns&domain product (#909)
scaleway/docs-content: faq/domains-and-dns.mdx

## What are the responsibilities of the domain name holders?

The domain name holders must: 

- comply with the terms and conditions posted by their Registrar, including applicable policies from their Registrar, the Registry and ICANN.
- review their registrar's current registration agreement, along with any updates.
- assume sole responsibility for the registration and use of their domain name.
- provide accurate information for publication in directories such as WHOIS, and promptly update this to reflect any changes.
- respond to inquiries from their Registrar within fifteen (15) days, and keep their Registrar account data and their payment information up-to-date. 

## Can I modify my domain name once it has been registered?

It is not possible to change domain name once the registration has been carried out. You can of course make a new registration with the correctly spelled domain name. 

## How long is the registration of a domain name valid for?

Your reservation is valid for a minimum of one year. You can reserve certain extensions for up to 10 years.  

## What  do I do if the domain name I want to register is not available?

> If a domain name is unavailable, you can test a light variation on the name desired. You can also add a letter, or a number. As a last resort, contact us(lien).

Author: If a domain name is unavailable, you can test a light variation on the name desired. You can also add a letter or a number. If you still encounter any issues, you can contact the [support team](https://console.scaleway.com/support/tickets).

> If a domain name is unavailable, you can test a light variation on the name desired. You can also add a letter, or a number. As a last resort, contact us(lien).

Author: ```suggestion
If a domain name is unavailable, you can test a light variation on the name desired. You can also add a letter or a number. If you still encounter any issues, you can contact the [support team](https://console.scaleway.com/support/tickets).
```

## Is domain renewal automatic?

No. You must renew your domain name before the end of its validity period.
If you choose the “auto-renew” feature, your domain name will be automatically renewed. 

## Do you send reminders on approach of expiry?

You will be notified by email 60 days before your domain name expires. Your domain name will then switch to “late renew” status. 

## Why has my current registrar opposed my request for change of domain name registrar?

The reasons for refusal are generally the following:
- The domain name is locked by a status “client transfer prohibited”: the applicant forgot to remove the padlock from the registrar before requesting the transfer of domain name
- The domain name was registered for less than 60 days
- The domain name was the subject of a previous transfer for less than 60 days
- The domain name has expired
- The domain name is the subject of a dispute in progress (udrp procedure for example)
- The owner notified his refusal to transfer the domain name to another registrar

## How long does a domain name transfer take?

> A domain name transfer usually takes between 1 to 10 days. A domain name transfer can however take longer.

Reviewer: ```suggestion
A domain name transfer usually takes between 1 to 10 days. However, a domain name transfer can take longer.
```

## Which extensions are available during the Public beta? 

There are seven extensions available:
- `.com` 
- `.eu`
- `.fr`
- `.net` 
- `.nl`
- `.org`
- `.pl` 

<Message type="note">
  To be able to use the `.eu` extension, you need to be a citizen of a country in the European Union. 
  To be able to use the `.fr` extension, you need to have an address in the European Economic Area and/or Switzerland.
</Message>

> ## Why is my domain registration refused? 

Reviewer: I'd say "rejected" or "denied" would be more idiomatic but maybe "refused" is the actual verb used in the console.

---
meta:

>   title: Domains & DNS

Reviewer 2: The final product name is « Domains **and** DNS », as the character `&` is not allowed anymore. I'm afraid you'll have to make a search and replace everywhere.

>   title: Domains & DNS

Reviewer 3: ```suggestion
  title: Domains and DNS
```

>   description: Discover Domains & DNS.

Reviewer 3: ```suggestion
  description: Discover Domains and DNS.
```

content:

>   h1: Domains & DNS

Reviewer 3: ```suggestion
  h1: Domains and DNS
```

>   paragraph: Discover Domains & DNS.

Reviewer 3: ```suggestion
  paragraph: Discover Domains and DNS.
```

image: DNS.webp
---

## What are the prerequisites to register a Domain name?

You need a valid and reliable email address and a valid debit or credit card. You must have a valid email address and make sure to keep your contact information updated during the whole period of registration.

## Which rights are associated with the owner of a domain?

> The owner of a domain can update all the technical and administrative parameters referring to their domain name, indicate and change the administrative, technical and invoicing contacts. During the public beta, you must contact the support team to modify the contacts you have entered. 

Reviewer 2: There is no "invoicing contact" in our product (because we actually are this contact on client behalf, and we invoice them in turn using our billing system)

## Do you send reminders on approach of expiry?

You will be notified by email 60 days before your domain name expires. Your domain name will then switch to “late renew” status. 

## Why has my current registrar opposed my request for change of domain name registrar?

The reasons for refusal are generally the following:
- The domain name is locked by a status “client transfer prohibited”: the applicant forgot to remove the padlock from the registrar before requesting the transfer of domain name
- The domain name was registered for less than 60 days
- The domain name was the subject of a previous transfer for less than 60 days
- The domain name has expired
- The domain name is the subject of a dispute in progress (udrp procedure for example)
- The owner notified his refusal to transfer the domain name to another registrar

## How long does a domain name transfer take?

A domain name transfer usually takes between 1 to 10 days. A domain name transfer can however take longer.

## Which extensions are available during the Public beta? 

There are 7 extensions available:
- `.com` 
- `.eu`
- `.fr`
- `.net` 
- `.nl`
- `.org`
- `.pl` 

<Message type="note">

> To be able to use the `.eu` extension, you need to be a resident of the European Union. 

Reviewer 2: for the `.eu`, you need to be a citizen of a country of the EU
for the `.fr`, you need to have an address in the European Economic Area + Switzerland

> To be able to use the `.eu` extension, you need to be a resident of the European Union. 

Reviewer 3: ```suggestion
To be able to use the `.eu` extension, you need to be a citizen of a country in the European Union. 
To be able to use the `.fr` extension, you need to have an address in the European Economic Area and/or Switzerland.

```

</Message>

## Why is my domain registration refused? 
Your domain registration may be refused if you do not meet the conditions set by the registry. 

> Ex: If your extension is “.fr” you must be a resident in France, otherwise your domain registration will be refused. 

Reviewer 2: see above

> Ex: If your extension is “.fr” you must be a resident in France, otherwise your domain registration will be refused. 

Reviewer 3: ```suggestion
Ex: If your extension is “.fr” you must have an address in the European Economic Area and/or Switzerland, otherwise your domain registration will be refused. 
```

---
meta:
  title: Domains and DNS
  description: Discover Domains and DNS.
content:
  h1: Domains and DNS
  paragraph: Discover Domains and DNS.
image: DNS.webp
---

## What are the prerequisites to register a Domain name?

You need a valid and reliable email address and a valid debit or credit card. You must have a valid email address and make sure to keep your contact information updated during the whole period of registration.

## Which rights are associated with the owner of a domain?

> The owner of a domain can update all the technical and administrative parameters referring to their domain name, indicate and change the administrative, technical and invoicing contacts. During the public beta, you must contact the support team to modify the contacts you have entered. 

Author: ```suggestion
The owner of a domain can update all the technical and administrative parameters referring to their domain name, indicate and change the administrative and technical and contacts. During the public beta, you must contact the support team to modify the contacts you have entered. 
```
